core/domain/collection_domain_test.py

```diff
@@ -641,9 +619,33 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
 tags: []
 title: A title
 """)
+    YAML_CONTENT_V3 = ("""category: A category
+language_code: en
+nodes:
+- acquired_skills:
+  - 0
+  - 1
+  exploration_id: Exp1
+  prerequisite_skills: []
+objective: ''
+schema_version: 2
+skills:
+  0:
+    skill_id: 0
```

Comment:
yep, we should have a validate function eventually. 

oh, that is true. I was that the skill object ought to have skill ID inside when I made this, but I realize we can just initialize the skill id, without the need to store it here.